server-spi-private/src/main/java/org/keycloak/models/LDAPConstants.java

```diff
@@ -82,6 +82,8 @@
     // Custom user search filter
     public static final String CUSTOM_USER_SEARCH_FILTER = "customUserSearchFilter";
 
+    public static final String TLS = "tls";
```

Comment:
Is it possible to rename this constant to USE_STARTTLS to make it more clear what does it mean?